stripe/api_resources/source.py

```diff
@@ -33,6 +35,7 @@ def detach(self, idempotency_key=None, **params):
                 "id",
             )
 
+    # DEPRECATED: use Source.list_source_transactions instead.
```

Comment:
Nit: users would not see this unless they go through the trouble of reading the library's source code. The idiomatic way of marking deprecated functions in Python is to use the `warnings` module:

```python
import warnings

class Source(...):
    ...

    # TODO: remove this in next major version
    def source_transactions(self, **params):
        warnings.warn(
            "source_transactions is deprecated, use list_source_transactions instead",
             DeprecationWarning
        )
        return self.request(...)
```

We could also add a Python docstring to display the deprecation in most IDEs:

```python
    def source_transactions(self, **params):
        '''source_transactions is deprecated, use list_source_transactions instead
        '''
        ...
```

Reply:
Gotcha – will definitely use a docstring then. Thanks!
I'd lean away from a warning (feels like overkill, users will see the changelog when it comes time to do a major upgrade, and it'll be an easy transition) unless you feel strongly the other way?

Reply:
Works for me 👍

Reply:
Please do add an internal comment with a `TODO` so we can grep for it and remember to remove it in the next major version!

Reply:
Hmm, would we not grep for "deprecated"?